emission/tests/modellingTests/modellingTestAssets.py

```diff
@@ -23,18 +23,26 @@ def generate_nearby_random_points(ref_coords,threshold):
     #convert given threshold in m to approx WGS84 coord dist.
     thresholdInWGS84 = threshold* (0.000001/0.11)
     
-    #generate a random coordinate in threshold's limit around the ref points. 
-    dx=random.uniform(-thresholdInWGS84/2,thresholdInWGS84/2)
-    dy=random.uniform(-thresholdInWGS84/2,thresholdInWGS84/2)
+    #generate a random coordinate in threshold's limit around the ref points. OR we 
 
+    # for eg, ref point is 0,0 and threshold is  100m , so we generate a radius from 0 to 50, say 34 
+    # in this example. A random radius is also generted from 0 to 360,say 0. We then take 34 step along x axis direction 
+    # till radius length to get our new point, (34,0). When this function is called the next time to generate a point
+    #that has to be binned with previous one, we again generate r and theta , say 24 , 180 this time. 
+    # Now this new point is at (-24,0). Both these points are within threshold (100 in this case)limit and therefore will 
+    #be binned together.
+    radius=random.uniform(0,thresholdInWGS84/2)
+    theta=random.uniform(0,2*math.pi)
+    dx = radius * math.cos(theta)
+    dy = radius * math.sin (theta)
     #This basically gives a way to sample a point from within a square of length thresholdInWGS84 
     # around the ref. point.  
```

Comment:
nit: **(future fix)**  fix the comment to reflect "the circle" instead of "the square"

Reply:
fixed.